src/Morph/morphTarget.ts

```diff
@@ -277,9 +277,10 @@ export class MorphTarget implements IAnimatable {
     /**
      * Creates a new target from serialized data
      * @param serializationObject defines the serialized data to use
+     * @param scene defines the hosting scene
      * @returns a new MorphTarget
      */
-    public static Parse(serializationObject: any): MorphTarget {
+    public static Parse(serializationObject: any, scene: Scene): MorphTarget {

```

Comment:
Maybe make it optional and check that the scene exists before starting the animation?